### PR TITLE
Adding Some Development Testing Documentation.

### PR DESCRIPTION
**Pull request by gmcquillan**
- Basic docs for how to run unit and e2e tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,3 +136,18 @@ kubectl delete --ignore-not-found thirdpartyresource \
 ```
 
 **The Prometheus Operator collects anonymous usage statistics to help us learning how the software is being used and how we can improve it. To disable collection, run the Operator with the flag `-analytics=false`**
+
+## Development
+
+### Prerequisites
+
+- golang environment
+- minikube (optional)
```
Comment (brancz) [inline README.md]: `make crossbuild` also requires a docker host as it uses docker under the hood to compile

**Comment by gmcquillan** [comment]
Kk. I'll update.

Unrelated note -- but I can't seem to make the unit tests pass on
origin/master currently. Is that a known state or am I missing something
locally?

On Mon, Aug 7, 2017 at 10:34 AM, Frederic Branczyk <notifications@github.com

**Comment by brancz** [comment]
hmm odd, `make test` works for me (using go 1.8.3)

**Comment by gmcquillan** [comment]
I'll open a separate issue with context, then.

**Comment by brancz** [comment]
Thanks!

**Pull request merged**